### ISSUE TITLE
Food likes/dislikes

### ISSUE THREAD

**Issue by SilviaKo** [ralfhe/FoReWeb#5]
Could we implement Food likes and dislikes?

**Comment by ralfhe** [comment]
Yes. This is a feature we definitively need and which should be implemented but most certainly by somebody after me.
